docs/en/manuals/input-gamepads.md

```diff
@@ -149,6 +147,14 @@ If your game is running from inside an `iframe` you must also make sure that the
 <iframe allow="gamepad"></iframe>
 ```
 
+### Standard gamepad
+(From Defold 1.3.8)
```

Comment:
1.3.8?

Reply:
Erm, I forgot that we upped the version!

Reply:
But do you really want to release it in current beta?

Reply:
Oh my god, I am so confused. It should be in the next release!

Reply:
1.4.1